### PR TITLE
Removed Bootstrap style link (version 3.3.7) and changed some elements styling

### PR DESCRIPTION
**Pull request by ghost**
I have removed Boostrap style link (version 3.3.7) and added the style link of the 4.0.0 version.
changed some elements styling (issue  #7 )

### CODE REVIEW

File: templates/screen_test.html [hunk 1]
```diff
@@ -1,27 +1,25 @@
...
 .container{
-  padding-top:120px;
+  margin-top:120px;
 }
-fieldset{
-  padding-top:40px;
-}
-button{
-  margin-bottom:50px;
+.screen-form {
```
Comment (tulikavijay) [inline templates/screen_test.html:7]: Can i see what it looks like. You can attach a screenshot

File: templates/register.html [hunk 2]
```diff
@@ -26,15 +29,13 @@
 {% block content %}
 <div class="container">
 {% if registered %}
-<div class="margins">
-<div class="heading">Thankyou for registering.</div>
-<div class="sub-heading"><a href={% url 'home' %}>Return to the home page</a></div>
+<h2>Thank you for registering.</h2>
```
Comment (tulikavijay) [inline templates/register.html:33]: I can't comprehend the size of the text, a screenshot would be great if possible for you.

File: templates/navbar.html [hunk 3]
```diff
@@ -1,8 +1,9 @@
-<nav class="navbar navbar-toggleable-md navbar-inverse fixed-top bg-inverse">
-     <button class="navbar-toggler navbar-toggler-right" type="button" data-toggle="collapse" data-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation">
+<nav class="navbar navbar-expand-lg navbar-inverse fixed-top navbar-dark bg-dark">
```
Comment (tulikavijay) [inline templates/navbar.html:1]: Can you explain the changes?

**Review by tulikavijay — COMMENTED** [review]
Hey! i have added some comments for you. Please check it out.

File: templates/screen_test.html (hunk 1 above)
Comment (ghost) [reply templates/screen_test.html:7]: screen-form_1

screen-form

File: templates/register.html (hunk 2 above)
Comment (ghost) [reply templates/register.html:33]: register

File: templates/navbar.html (hunk 3 above)
Comment (ghost) [reply templates/navbar.html:1]: Hi @tulikavijay, 
The responsive navbar toggle has been updated in bootstrap 4, we can specify the background color using bg- classes: `bg-light`, `bg-dark`, etc. and for _lighter_ text color we can use `navbar-dark`.

**Comment by tulikavijay** [comment]
Everything looks great! 
One thing i see though is the submit button on register page, it looks slightly disproportionate. I think that might be because of bootstrap4? Anyhow if you can fix it, it will be awesome. I'll merge your PR as soon as your done.

**Comment by ghost** [comment]
Hi @tulikavijay 
How about this fix?
register

Also, I saw that the login button is not styled. Can I make the change?
login

**Comment by tulikavijay** [comment]
This fix looks good 👍 
Also sure you can make changes to the login page.

**Pull request merged**